src/include/processor/operator/persistent/reader/csv/base_csv_reader.h

```diff
@@ -73,11 +78,18 @@ class BaseCSVReader {
 
     common::block_idx_t currentBlockIdx;
 
+    uint64_t rowNum;
```

Comment:
Do you mean the total number of rows or current row number?
If it is current row number, i suggest renaming to curRowIdx.

Reply:
updated